### PR TITLE
Leaf 3546 - API code consistency

### PR DESCRIPTION
**Pull request by jampaul3**
This just renames 2 API endpoints.

### CODE REVIEW

**Review by maxrichardjr — APPROVED** [review]

**Review by raphaelfretter — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/templates_history/template_editor/1685036655_view_homepage.tpl
[the remark this answers is not in the document]
Comment (jampaul3) [reply LEAF_Request_Portal/templates_history/template_editor/1685036655_view_homepage.tpl:1]: no, hence the reason we need the LEAF_Request_Portal/templates_history/* in the gitignore file. We don't want these to get pushed to anything. These were just created in my testing.

**Review by pete-nerantzinis — APPROVED** [review]
Approving but please test this since there are so many changes we might even need to do a search on the reportTemplates as there could be custom code with those api calls.

**Review by jrodcode — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by AmandaSumner — APPROVED** [review]

**Review by aerinkayne — APPROVED** [review]
Just noting other comments about gitignore for template hsitory tracking (and custom override templates too if it does not cause issues)

**Comment by mgaoVA** [comment]
@jampaul3 For API renames:

1. The old API should remain active should remain until we can show that it hasn't been used in the past 30 days
2. While the old API is active, there should be a comment to indicate its deprecation status, and reference the new API

If we find that its being used after we stop using it ourselves, we'll need to reach out to the site admins still using it and request that they update their customizations.

**Review by maxrichardjr — APPROVED** [review]

**Pull request merged**